src/jit/liveness.cpp

```diff
@@ -1103,6 +1103,13 @@ void Compiler::fgExtendDbgLifetimes()
             {
                 continue;
             }
+#if !defined(_TARGET_64BIT_) && !defined(LEGACY_BACKEND)
+            // On x86, this applies to long-typed variables as well once we've hit LIR. 
```

Comment:
I don't think that's correct. There are two points at which we run liveness when targeting the RyuJIT backend:
- Before building SSA
- After lowering

The latter is also after decomposition.

I had hoped to be able to skip these because the decomposition of a `st.lclVar` node (which is a complete def) into two `st.lclFld` nodes (which are each a partial def but comprise a complete def when taken together) causes the liveness to go funny (since the `st.lclFld`s are also treated as uses). We might have to rethink this sort of decomposition after all.